src/applications/vaos/new-appointment/components/VAFacilityPage/SingleFacilityEligibilityCheckMessage.jsx

```diff
@@ -15,23 +18,48 @@ export default function SingleFacilityEligibilityCheckMessage({
   if (!eligibility.requestPastVisit) {
     message = (
       <>
-        However, in order to request an appointment at this facility for the
-        chosen type of care, you need to have been seen within the past{' '}
-        {eligibility.requestPastVisitValue} months.
+        <p>
+          To request an appointment online at this location, you need to have
```

Comment:
Same comment as above, this is technically inaccurate because it's not any visit to the facility. I think there's a separate issue for it, but if we're changing the text now, we should probably take it into account.